.github/workflows/release-java.yml

```diff
@@ -0,0 +1,23 @@
+name: Release Java
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  release-java:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2.3.4
+
+      - name: Set up JDK 11 for running Gradle
+        uses: actions/setup-java@v2
+        with:
+          distribution: adopt
+          java-version: 11.0.11+9
+
+#      - name: Increment Patch Version
+#        run: ./gradlew incrementPatchVersion
+
+      - name: Publish
+        run: ./gradlew publish --dry-run
```

Comment:
why dry run? We're still evaluating this?

Reply:
cool, don't you want to add to the dry run the incrementPatchVersion as well? It's comment out above

Reply:
Actually you're right, didn't know dry-run works on scripts so just added that :)